source/portable/NetworkInterface/Zynq/NetworkInterface.c

```diff
@@ -327,7 +331,7 @@ static BaseType_t xZynqNetworkInterfaceInitialise( NetworkInterface_t * pxInterf
             uint32_t ulValue = XEmacPs_ReadReg( pxEMAC_PS->Config.BaseAddress, XEMACPS_NWCFG_OFFSET );
             /* Allow the use of hashed MAC addresses. */
             ulValue |= XEMACPS_NWCFG_MCASTHASHEN_MASK;
-            #warning As 'MCASTHASHEN' doesn't seem to work, use the promiscuous mode so that IPv6 multicast packets are received.
+            /* As 'MCASTHASHEN' doesn't seem to work, use the promiscuous mode so that IPv6 multicast packets are received. */
```

Comment:
Can we keep this warning inside the new `ipconfigPORT_SUPPRESS_WARNING` config check? And remove the the apostrophe in word `doesn't`.

Reply:
I don't think it's valid to keep this warning message always printing. Any warning message should be handled by configurations. And I don't think it's valid to warn users about this.